examples/data/cleansed/metadata.yml

```diff
@@ -0,0 +1,61 @@
+scada:
+  time: time
+  id: id
+  WTUR_W: WTUR_W
+  WMET_HorWdSpd: WMET_HorWdSpd
+  WMET_HorWdDir: WMET_HorWdDir
+  status: status
+  WROT_BlPthAngVal: WROT_BlPthAngVal
+  WMET_EnvTmp: WMET_EnvTmp
+  energy: energy
+  frequency: 10T
+meter:
+  time: time
+  MMTR_SupWh: MMTR_SupWh
+  frequency: 10T
+tower:
+  time: time
+  id: id
+  frequency: 10T
+status:
```

Comment:
I'm not sure if -25 has tag names for status codes. We could leave this as is for now.

Reply:
That works for me, we've still got ample time to adopt anything else later on.